coap-core/src/main/java/com/mbed/coap/server/CoapServerBuilder.java

```diff
@@ -37,6 +37,9 @@
  */
 public abstract class CoapServerBuilder<T extends CoapServerBuilder> {
     private static final int DEFAULT_MAX_DUPLICATION_LIST_SIZE = 10000;
+    private static final int DEFAULT_DUPLICATE_DETECTOR_CLEAN_INTERVAL = 10000;
```

Comment:
Should this have _MILLIS at the end too?

Reply:
Good catch! I fixed that.